dist/40-10007-schema.json

```diff
@@ -297,11 +297,6 @@
       "type": "string",
       "pattern": "^\\d{8,9}$"
     },
-    "militaryServiceNumber": {
```

Comment:
there's an ongoing discussion about editing this area in platform - I'll link you